src/open_inwoner/conf/parts/maileditor.py

```diff
@@ -290,16 +290,76 @@
             },
         ],
     },
-    "case_notification": {
-        "name": _("Case update notification"),
+    "case_status_notification": {
+        "name": _("Case status update notification"),
         "description": _(
-            "This email is used to notify people an update happened to their case"
+            "This email is used to notify people about a new status being set on their case"
         ),
         "subject_default": "Update to your case at {{ site_name }}",
         "body_default": """
             <p>Beste</p>
 
-            <p>You are receiving this email because one of your cases received a new status update or document attachment.</p>
+            <p>You are receiving this email because one of your cases received a new status update.</p>
+
+            <table>
+                <tr>
+                    <th>Case identification</th>
+                    <td>{{ identification }}</td>
+                </tr>
+                <tr>
+                    <th>Case type</th>
+                    <td>{{ type_description }}</td>
+                </tr>
+                <tr>
+                    <th>Start date</th>
+                    <td>{{ start_date }}</td>
+                </tr>
+            </table>
+
+            <p><a href="{{ case_link }}">Go to your case</a> </p>
+
+            <p>Met vriendelijke groet,
```

Comment:
This is Dutch, the previous text is English. I suggest the entire default message should be Dutch, since it is client-facing.